media-source-respec.html

```diff
@@ -125,7 +125,7 @@ <h2>
       </p>
       <figure>
         <a href=
-        'https://w3c.github.io/media-source/pipeline_model_description.html#pipelinedesc'><picture><img src="pipeline_model.svg"
```

Comment:
FWIW, the spec-prod action used not to support automatic publication of local HTML assets linked from a spec, and the absolute URL was thus needed (the file does not exist under www.w3.org/TR yet). But that's no longer the case, that feature got added to spec-prod 3 years ago (and I of course forgot to update the URL in MSE accordingly!).

Reply:
The PR now adds this description to the spec and removes the separate linked page.